tests/tt_eager/python_api_testing/unit_testing/misc/test_sharded.py

```diff
@@ -988,6 +988,96 @@ def test_sharded_matmul_2d(
     assert passing
 
 
+@pytest.mark.parametrize("in0_sharded", [True, False], ids=["in0_sharded", "in0_unsharded"])
+@pytest.mark.parametrize("in1_sharded", [True, False], ids=["in1_sharded", "in1_unsharded"])
+@pytest.mark.parametrize("out_sharded", [True, False], ids=["out_sharded", "out_unsharded"])
+@pytest.mark.parametrize("activations_dtype", [ttl.tensor.DataType.BFLOAT16, ttl.tensor.DataType.BFLOAT8_B])
```

Comment:
Can we sweep `output_dtype` as well and not have it default to `activations_dtype`?